pyproject.toml

```diff
@@ -12,7 +12,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "openassetio"
 version = "1.0.0a5"
-requires-python = ">=3.9"
+requires-python = ">=3.7"
```

Comment:
[consider][minor] Do we want to update the `classifiers` below too?

Reply:
Good spot, thanks, yes. Added 3.7 [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/694/commits/488c8ad2c31e511c3e946731d0eb5e7e7b658546).